src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -598,11 +470,11 @@ export default class ReviewConfirmDissolution extends Mixins(DateMixin) {
   background-color: $gray1;
 }
 
-.section-container {
+::v-deep .section-container {
   padding: 1.5rem 1.5rem;
 }
 
-.review-header {
+::v-deep .review-header {
```

Comment:
This v-deep should not be required. Class "review-header" is in this component (not a sub-component), and you didn't change that, and I think it used to work.

Reply:
I believe I needed this because the new sub-component(`CompleteResolutionsSummary` needs it.  

![image](https://user-images.githubusercontent.com/6685223/143601966-bbd41e35-4e9d-4c4e-95e8-7f4d4ab2d6bc.png)

Reply:
So both ReviewConfirmDissolution.vue and CompleteResolutionSummary.vue use the same class? Is it easy to rename one of them to avoid conflicts?